pkgs/development/ocaml-modules/tezos/protocol-demo-noops.nix

```diff
@@ -0,0 +1,18 @@
+{ lib
+, buildDunePackage
+, tezos-stdlib
+, tezos-protocol-compiler
+}:
+
+buildDunePackage {
+  pname = "tezos-protocol-demo-noops";
+  inherit (tezos-stdlib) version src useDune2 preBuild doCheck;
+
+  propagatedBuildInputs = [
+    tezos-protocol-compiler
```

Comment:
Does it need to be propagated?

Reply:
Yes, I think the name is misleading, it's just a library